### ISSUE TITLE
Every editor is way too zoomed in at first

### ISSUE THREAD

**Issue by notexplosive** [defold/editor2-issues#2971]
Maybe I'm doing something wrong but I'm finding that every time I open an editor (say: new -> game object) the editor is zoomed in wayyyy more than is at all practical. Example:

- Right click assets pane: new game object
- Add a component -> sprite
- attach image from tilesource, image is a 32x32 icon
- My view is now a hyper-zoomed in view of the middle 4-8ish pixels of the image, I have to zoom out bunch to even get a sense of what the object even looks like.

Every editor has this problem. It's especially annoying with the tile map editor because I'm zoomed in to the size of one tile, so I need to zoom way out before I can even start building my map.

Quick workaround exists for some editors: pressing F to center view on the object (can this happen automatically upon adding the component/game object? maybe as a setting?)

I get that it's hard to predict how big the user wants the view to be, but if I'm having this issue with 32x32 icons I can't imagine how bad this is for people working in non-pixel-art resolutions. 

Sorry if this is a dup but I can't seem to find anyone else reporting this (aside from the tutorial mentioning that this might be a problem). Maybe my 4K monitor makes this worse? I have my UI scaling set to 150%, could that be a factor?

Thanks!

**Comment by subsoap** [comment]
I agree about this, I usually have to select something and then press F / always zoom out.

There are problems too when including particlefx that causes the editor to zoom out way more than it should when opening a file.
